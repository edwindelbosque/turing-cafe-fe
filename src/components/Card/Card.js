import React from 'react';
import './Card.scss';

const Card = () => {
  return (
    <p>card</p>
  )
};

export default Card;