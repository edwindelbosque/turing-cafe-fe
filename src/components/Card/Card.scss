.card {
  padding: 10px;
  background: #FFFFFF;
  border-radius: 10px;
  border: 5px solid #008080;
  width: 250px;
  height: 200px;
  display: flex;
  flex-direction: column;
  justify-content: space-around;
  color: #008080;
  align-items: center;
  font-size: 14px;

  button {
    background: #008080;
    height: 30px;
    width: 90%;
    color: white;
    font-size: 18px;
    transition: .15s;

    &:hover {
      background: darken(#008080, 8%);
      transition: .15s;
    }

    &:active {
      background: darken(#008080, 14%);
      transition: .15s;
    }

    &:focus {
      outline: none;
    }
  }
}