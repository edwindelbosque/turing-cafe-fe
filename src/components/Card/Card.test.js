import React from 'react';
import Card from './Card';
import { shallow } from 'enzyme';

describe('Card', () => {
  it('Should match the snapshot', () => {
    const cancelReservation = jest.fn();
    const wrapper = shallow(
      <Card
        key={1}
        id={1}
        name={'Charli'}
        date={'10/23'}
        time={'12:00 PM'}
        number={'2'}
        cancelReservation={cancelReservation}
      />
    );

    expect(wrapper).toMatchSnapshot();
  })
})