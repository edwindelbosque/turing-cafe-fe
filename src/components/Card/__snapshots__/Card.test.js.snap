// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Should match the snapshot 1`] = `
<div>
  <h2>
    Charli
  </h2>
  <h3>
    10/23
  </h3>
  <p>
    12:00 PM
  </p>
  <p>
    2
  </p>
  <button
    onClick={[Function]}
  >
    Cancel Reservation
  </button>
</div>
`;
